### ISSUE TITLE
Why BCE is used instead of CE with Softmax? 

### ISSUE THREAD

**Issue by JoyHuYY1412** [brjathu/iTAML#11]
Each task seems to be a multi-class classification, so why not using nn.CrossEntropyLoss?

**Comment by brjathu** [comment]
To minimize catastrophic forgetting, it's better not to pull down the previously learned distributions, even if they are not present in the training data.
